Dockerfile

```diff
@@ -91,6 +96,9 @@ COPY requirements /tmp/requirements
 RUN set -x \
     && if [ "$DEVEL" = "yes" ]; then pip --no-cache-dir --disable-pip-version-check install -r /tmp/requirements/dev.txt; fi
 
+RUN set -x \
+    && if [ "$IPYTHON" = "yes" ]; then pip --no-cache-dir --disable-pip-version-check install -r /tmp/requirements/ipython.txt; fi
```

Comment:
Can we add a condition on `$DEVEL` here also to ensure this doesn't accidentally get set in a deployable asset?

`if [ "$IPYTHON" = "yes" ] &&  "$DEVEL" = "yes" ]; then ...`

Reply:
Totally, I'll do that in the next couple days. I just wanted to get this replacement solution out there so I didn't let it languish too long.